src/Components/LeftSidebar/LeftSidebar.module.css

```diff
@@ -78,40 +85,120 @@ a {
 }
 
 .small_profile {
-    display: flex;
-    align-items: center;
-    justify-content: flex-start;
-    color: var(--content-color);
-    padding: 1rem;
-    margin: auto 0 1rem 1rem;
-    border-radius: 100rem;
-    cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  color: var(--content-color);
+  padding: 1rem;
+  margin: auto 0 1rem 1rem;
+  border-radius: 100rem;
+  cursor: pointer;
 }
 
 .small_profile:hover {
-    background-color: var(--faint-bg);
-    transition: 0.2s ease-in-out;
+  background-color: var(--faint-bg);
+  transition: 0.2s ease-in-out;
 }
 
 .avatar {
-    height: 5rem;
-    width: 5rem;
-    border-radius: 50%;
+  height: 5rem;
+  width: 5rem;
+  border-radius: 50%;
 }
 
 .small_profile h1 {
-    font-size: 2rem;
-    margin-left: 2rem;
-    font-weight: 600;
+  font-size: 2rem;
+  margin-left: 2rem;
+  font-weight: 600;
 }
 
 .small_profile h3 {
-    font-size: 2rem;
-    margin-left: 2rem;
-    font-weight: 500;
-    color: var(--faint-text-clr);
+  font-size: 2rem;
+  margin-left: 2rem;
+  font-weight: 500;
+  color: var(--faint-text-clr);
 }
 
 .user {
-    align-self: flex-start;
-}
\ No newline at end of file
+  align-self: flex-start;
+}
+
+.mobile_icon_container span.mobile_close {
+  /* display: none; */
+  color: var(--content-color);
+  cursor: pointer;
+  margin-left: 1rem;
+}
+
+.mobile_icon_container span.mobile_close:hover {
+  color: var(--primary-color);
+  transition: 0.2s ease-in-out;
+}
+
+@media screen and (max-width: 750px) {
+  .left_sidebar {
+    position: fixed;
+    width: 30rem;
+    background-color: var(--default-color);
+    z-index: 2;
+    transform: translateX(-100%);
+    transition: 0.2s ease-in-out;
+  }
+
+  .left_sidebar_toggle {
+    position: fixed;
+    width: 30rem;
+    background-color: var(--default-color);
+    z-index: 2;
+    transform: translateX(0%);
+    transition: 0.2s ease-in-out;
+    grid-area: left-sidebar;
+    top: 0;
+    border-right: 1px solid var(--content-color);
+    padding-right: 2rem;
```

Comment:
Suggestion: instead of padding right and left use shorthand syntax.
i.e padding: 0 2rem 0 1rem;

Reply:
ok will make the changes